plugins/webp-uploads/picture-element.php

```diff
@@ -168,3 +168,26 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		$image
 	);
 }
+
+/**
+ * Adds CSS to make Lightbox trigger BUTTON visible on an Image lightbox when Picture Element is enabled.
+ *
+ * @since n.e.x.t
+ */
+function webp_make_lightbox_trigger_button_visible(): void {
+	if ( has_filter( 'render_block_core/image', 'block_core_image_render_lightbox' ) ) {
```

Comment:
Please review the code linting failures, see https://github.com/WordPress/performance/actions/runs/12910976968/job/36002299388?pr=1814.
```suggestion
	if ( (bool) has_filter( 'render_block_core/image', 'block_core_image_render_lightbox' ) ) {
```